src/data/polynomial/degree/to_trailing_degree.lean

```diff
@@ -0,0 +1,51 @@
+import tactic
+import data.polynomial.degree.basic
+import data.polynomial.degree.to_basic
+import data.polynomial.degree.trailing_degree
+
+open polynomial finsupp finset
+
+namespace polynomial
+variables {R : Type*} [semiring R] {f : polynomial R}
+
+@[simp] lemma trailing_coeff_eq_zero : trailing_coeff f = 0 ↔ f = 0 :=
+⟨λ h, by_contradiction $ λ hp, mt mem_support_iff.1
+  (not_not.2 h) (mem_of_min (trailing_degree_eq_nat_trailing_degree hp)),
+λ h, h.symm ▸ leading_coeff_zero⟩
+
+lemma trailing_coeff_nonzero_of_nonzero : f ≠ 0 ↔ trailing_coeff f ≠ 0 :=
+not_congr trailing_coeff_eq_zero.symm
+
+lemma nat_trailing_degree_mem_support_of_nonzero : f ≠ 0 → nat_trailing_degree f ∈ f.support :=
+begin
+  intro,
```

Comment:
```suggestion
  intro a,
```
Writing the name of the variable makes it easier for someone just reading the source code to see where `a` came from later on.